plugins/linux-pipewire/pipewire.h

```diff
@@ -27,8 +27,11 @@
 
 typedef struct _obs_pipewire_data obs_pipewire_data;
 
+extern const struct pw_stream_events stream_events;
+
```

Comment:
Please don't. **If** allowing plugins to override stream events is a good idea - something I'm not convinced of yet - then it's better to write it as follows:

```c
struct pw_stream_events obs_pipewire_get_default_stream_events (void);
```

then in the C file it returns a copy of the structure:

```c
struct pw_stream_events obs_pipewire_get_default_stream_events (void)
{
	return stream_events;
}
```

Doing this will at least prevent plugins from accidentally screwing up other plugins.